.github/workflows/coverage_runner.yml

```diff
@@ -0,0 +1,99 @@
+name: Run tests, measure coverage and upload results to codecov
+
+on:
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: Enter PR number coming from community to run coverage on it. Do not run for PR coming from hazelcast organization, they will be run automatically.
+        required: true
+  push:
+    branches:
+      - master
+      - '[45].*.x'
+  pull_request_target:
+    branches:
+      - master
+      - '[45].*.x'
```

Comment:
In python workflow we have a 
```
types:
      [opened]
```
here. Why don't we have it here and what is it for?

Reply:
No we don't. We had but I removed it because when setting the `[opened]` workflow is triggered only when it is opened. When you send new commits it is not triggered again.

>Note: By default, only the opened, synchronize, and reopened activity types trigger workflows that run on the pull_request event. To trigger workflows by different activity types, use the types keyword.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
Ok thanks.